front_end/sdk/DartSupport.js

```diff
@@ -34,6 +34,28 @@ Dart.ReloadActionDelegate = class {
   }
 };
 
+Dart.ToggleSourcemapsActionDelegate = class {
+
+  constructor() {}
+
+  /**
+   * @override
+   * @param {!UI.Context} context
+   * @param {string} actionId
+   * @return {boolean}
+   */
+  handleAction(context, actionId) {
```

Comment:
nit: this seems like a feature you could upstream. Perhaps use a generic name instead of a Dart name.